css/style.css

```diff
@@ -0,0 +1,360 @@
+  font-size: 20px;
+  border: 2px solid #999;
+  padding: 10px;
+}
+
+.footer {
+  width: 100%;
+  padding: 10px;
+  overflow: hidden;
+  background-color: #999;
+}
+
+.footer-logo {
+  max-width: 50px;
+  float: left;
+}
+
+.footer-return {
+  max-width: 40px;
+  float: right;
+}
+
+/* here is where the about me page starts */
+
+.about-me {
+  margin: 0 auto;
+  width: 70%;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  overflow: hidden;
+}
+
+.about-image {
+  border-radius: 50%;
+  max-width: 300px;
+  float: left;
+  display: block;
+  margin-right: 25px;
+}
+
+.personal-website-cta {
+  clear: both;
+  margin: 0 auto;
+  width: 40%;
+  padding: 75px;
+}
+
+.personal-website-cta a:link, .personal-website-cta a:active, .personal-website-cta a:visited {
+  color: #999;
+  font-size: 50px;
+  border: 2px solid #999;
+  border-radius: 10px;
+  padding: 10px;
+  text-decoration: none;
+}
+
+/* individual detail page */
+
+.primary-image-content {
+  width: 30%;
+  margin: 0 auto;
+  padding-top: 40px;
+}
+
+.primary-image {
+  max-width: 400px;
+}
+
+.top-line {
+  border-top: 2px solid #999;
+  margin-top: 100px;
+}
+
+.buy-now {
+  float: right;
+  color: #000;
+  margin-top: 30px;
+  margin-right: 30px;
+}
+
+.buy-now a:link, .buy-now a:visited, .buy-now a:active {
+  font-size: 20px;
+  border: 2px solid #000;
+  color: #000;
+  padding: 10px;
+  text-decoration: none;
+}
+
+@media only screen and (max-width: 768px) {
+
+  h1 {
+    font-size: 23px;
+  }
+
+  .navigation a:link, .navigation a:active, .navigation a:visited {
+    font-size: 18px;
+    margin: 10px;
+  }
+
+  .cta a:link, .cta a:active, .cta a:visited {
+    font-size: 20px;
+    margin: 100px;
+  }
+
+  .question {
+    padding: 20px;
+    float: none;
+    text-align: center;
+  }
+
+  .description {
+    padding: 20px;
+  }
+
+  .self {
+    float: none;
+    margin: 0 auto;
+    margin-bottom: 30px;
+  }
+
+  .shop {
+    width: 100%;
+    margin: 0 auto;
+    text-align: center;
+  }
+
+  .art {
+    float: none;
+    margin: 0 auto;
+    padding: 10px;
+  }
+
+  .see-all-art {
+    float: none;
+    padding-bottom: 30px;
+    padding-top: 30px;
+    padding-right: 0px;
+    margin: 0 auto;
+    text-align: center;
+  }
+
+  .about-image {
+    max-width: 250px;
+    float: none;
+    margin: 0 auto;
+    padding-bottom: 25px;
+  }
+
+  .personal-website-cta {
+    float: none;
+    margin: 0 auto;
+    width: 100%;
+    text-align: center;
+  }
+
+  .personal-website-cta a:link, .personal-website-cta a:active, .personal-website-cta a:visited {
+    font-size: 20px;
+  }
+
+  .about-me {
+    padding-top: 10px;
+    padding-bottom: 20px;
+  }
+
+  .primary-image-content {
+    width: 90%;
+    margin: 0 auto;
+    padding-top: 20px;
+  }
+
+  .primary-image {
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  .buy-now {
+    float: none;
+    padding-top: 10px;
+    padding-bottom: 30px;
+    margin: 0 auto;
+    text-align: center;
+  }
```

Comment:
Great job with the responsive bit - it really looks good when you shrink the browser down.